frontend/next-env.d.ts

```diff
@@ -3,4 +3,4 @@
 /// <reference types="next/navigation-types/compat/navigation" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
+// see https://nextjs.org/docs/app/building-your-application/configuring/typescript for more information.
```

Comment:
I believe this change occurred automatically when I started up a local server, not entirely sure why.

Reply:
I get the same. It is a doc link update so I think we can just add it in this PR.